backend/helpers/RefundEmailHelper.py

```
@@ -0,0 +1,701 @@
+        msg.attach(text_part)
+        msg.attach(html_part)
+        
+        # Send email
+        server = smtplib.SMTP(SMTP_SERVER, SMTP_PORT)
+        server.starttls()
+        server.login(SMTP_USERNAME, SMTP_PASSWORD)
+        server.send_message(msg)
+        server.quit()
+        
+        logging.info(f"Email sent successfully to {to_email}")
+        return {"success": True, "message": "Email sent successfully"}
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)